.github/dependabot.yml

```diff
@@ -11,6 +11,7 @@ updates:
   # Daily: Check minor and patch updates
   - package-ecosystem: "npm"
     directory: "/"
+    open-pull-requests-limit: 10
```

Comment:
Why do we need this?

Reply:
currently dependabot doesn't open any new PRs if the amount of open PRs by dependabot (currently 5) is even already reached. Especially within the Elements repo we might have some more PRs open, especially regarding major framework updates (and dependabot even also doesn't support grouping related dependencies so far, so the problem might even further increase), which would lead to that dependabot silently skips until the next cycle to analyse for new updates.
We might even also be able to deactivate this limit in total, but I still like the idea to at least limit it to around 10 or so, and work with a daily check for new updates.